src/Telnyx.Example/WDRReportsExample .cs

```diff
@@ -0,0 +1,177 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Telnyx.net.Entities;
+using Telnyx.net.Entities.Wireless.WDRReports;
+using Telnyx.net.Services.Wireless.WDRReports;
+
+namespace Telnyx.Example
+{
+    public class WDRReportsExample
+    {
+        private readonly WDRReportsService service = new WDRReportsService();
+
+        private const string id = "6a09cdc3-8948-47f0-aa62-74ac943d6c58";
+
+        public WDRReportsRecord Get()
+        {
+            WDRReportsRecord result = new WDRReportsRecord();
+
+            try
+            {
+                result = service.Get(id, null);
+                Console.WriteLine(JsonConvert.SerializeObject(result));
+            }
+            catch (TelnyxException ex)
+            {
+                Console.WriteLine("exception");
+                Console.WriteLine(JsonConvert.SerializeObject(ex));
+            }
+            return result;
+        }
+
+        public async Task<WDRReportsRecord> GetAsync()
+        {
+            WDRReportsRecord result = new WDRReportsRecord();
+
+            try
+            {
+                result = await service.GetAsync(id, null);
+                Console.WriteLine(JsonConvert.SerializeObject(result));
+            }
+            catch (TelnyxException ex)
+            {
+                Console.WriteLine("exception");
+                Console.WriteLine(JsonConvert.SerializeObject(ex));
+            }
+            return result;
+        }
+
+        public TelnyxList<WDRReportsRecord> List()
+        {
+            TelnyxList<WDRReportsRecord> result = new TelnyxList<WDRReportsRecord>();
+            ListWDRReportsOptions listOptions = new ListWDRReportsOptions
```

Comment:
Change to ListOptions, unless we plan to extend this list options object

Reply:
done.